src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2188,11 +2188,12 @@ def create_deploy_webapp(cmd, name, location=None, sku=None, dryrun=False):  # p
 
     # determine the details for app to be created from src contents
     lang_details = get_lang_from_content(src_dir)
+    print(lang_details)
     # we support E2E create and deploy for selected stacks, any other stack, set defaults for os & runtime
     # and skip deployment
     if lang_details['language'] is None:
         do_deployment = False
-        sku = sku | 'F1'
+        sku = sku or 'F1'
         os_val = OS_DEFAULT
         detected_version = '-'
         runtime_version = '-'
```

Comment:
How do we determine the runtime if there is no project file?

Reply:
@burkeholland what do you meanby project file?

Reply:
Like a package.json. I’m trying to determine if up requires the package.json to determine the runtime.